jooby/src/main/java/io/jooby/DefaultContext.java

```diff
@@ -228,10 +228,10 @@
   }
 
   @Override default @Nonnull String getRequestURL() {
-    return getRequestURL("");
+    return getRequestURL(null);
```

Comment:
This one add call `getRequestURL(getRequestPath())` and append queryString if present

Reply:
OK, this is probably a better idea, yes